build/testing/applitools/run-msft-docs.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+
+let spawn = require('child_process').spawnSync;
+
+/**
+ * Get Branch Name using child_process and Git which runs Asynchronously
+ */
+class Config {
+
+    constructor() {
+        this.DomainName = "https://msft-docs.fast-dna.net";

```

Comment:
since these are not classes, they should be lower case for single words or camelCase.